app/actions/views/channel.js

```diff
@@ -62,12 +64,16 @@ export function loadChannelsIfNecessary(teamId) {
     };
 }
 
-export function loadChannelsByTeamName(teamName) {
+export function loadChannelsByTeamName(teamName, error) {
     return async (dispatch, getState) => {
         const state = getState();
         const {currentTeamId} = state.entities.teams;
         const team = getTeamByName(state, teamName);
 
+        if (!team && error) {
+            error();
```

Comment:
I agree. I changed the one argument to `errorHandler` to avoid a variable naming conflict, but I too think it reads clearer than `callback` or `error`.

Changing all to same name `errorHandler`.